### DISCUSSION TITLE
ender 3 with 4.2.7 silent board and creality touch screen

### DISCUSSION THREAD

**Discussion by firemanz8** [Jyers/Marlin#948] [Q&A] [answered]
i bought the [V4.2.7 with TMC2225 Driver Marlin 2.0.1 for Ender 3](https://www.amazon.com/gp/product/B07TFTVTXK/ref=ppx_yo_dt_b_asin_title_o07_s00?ie=UTF8&th=1)  for my ender 3 and I have an ender 3 v2 that is running jyers firmware and I love it and I want to install it on my ender 3 but I bought that mainboard and then the reality touchy screen and when I go to update it gets stuck on the boot screen and I can't figure out how to get it to work I use the jyers 4.2.7 firmware

**Comment by smooreace** [comment]
I dont think you can use the touchscreen with Jyers... yet.

**Reply by firemanz8** [reply]
do you know when possibly he will add the feature

**Comment by smooreace** [comment, marked as answer]
Its probably going to be a side effect of moving away from the
Extensible_UI thought process and working back toward the main Marlin
display code. (I know that's not worded well at all, but I'm drawing a
total blank for the correct terminology ATM) I don't know that its so much
that he has to do anything special to be able to use it, its more about
getting back to the more official way of handling the display. And I would
even pretend to have any business defining or knowing Jyers timelines. The
dude gives away FAR too much of his time already!

**Reply by firemanz8** [reply]
true he puts a lot of time into this firmware and its amazing how he doesn't even get anything out of it

**Comment by smooreace** [comment]
He does... but not nearly enough! Since he started his Patreon page several
have tossed him a few $$
